src/mysky/index.ts

```diff
@@ -139,10 +139,10 @@ export class MySky {
     const hostDomain = await client.extractDomain(window.location.hostname);
     const permissions = [];
     if (skappDomain) {
-      // TODO: Are these permissions correct?
-      const perm1 = new Permission(hostDomain, skappDomain, PermCategory.Hidden, PermType.Read);
-      const perm2 = new Permission(hostDomain, skappDomain, PermCategory.Hidden, PermType.Write);
-      permissions.push(perm1, perm2);
+      const perm1 = new Permission(hostDomain, skappDomain, PermCategory.Discoverable, PermType.Write);
```

Comment:
Does this mean you only have write permissions on discoverable files and not read permission? Just asking because that seems weird to me.

Reply:
afaik discoverable read is not permissioned, it's available to read by everyone but I'll let Marcin chime in

Reply:
That's right, so that particular permission is redundant. Thanks @kwypchlo